### DISCUSSION TITLE
How to exclude a location or directory from authentication

### DISCUSSION THREAD

**Discussion by fong1999** [OpenIDC/mod_auth_openidc#933] [Q&A]
Hi! 

We have set up to use Azure to authenticate our site, which is working as expected. Now we have a scenario that we need to exclude a location or directory without authentication for a newrelic agent to retrieve the apache web server status.  How to achieve that?

For example,

What I am looking for is when newrelic agent opens this link `https://www.example.com/nr-server-status` without the authentication.

```
<Location "/nr-server-status">
SetHandler server-status
AuthType None

.......
</Location>

<Location />
    AuthType openid-connect
    Require valid-user
</Location>
```

I tried the following: 

1) Use `AuthType None` settings, but it does not work. 
2) Moved below to the top, like below. Then when I tried to go to this site `https://www.example.com/nr-server-status`, I got `Forbidden resource`. 

```

<Location />
    AuthType openid-connect
    Require valid-user
</Location>


<Location "/nr-server-status">
SetHandler server-status
AuthType None

.......
</Location>

````

However, nothing works yet. Any suggestions? 

Thanks,
Fong

**Comment by hmoffatt** [comment]
I think you need `require all granted` on the status location, not the AuthType clause.

**Reply by fong1999** [reply]
I will try that! Thank you!
